### PR TITLE
Add a timeout for housekeeping when using scylla with no external connectivity

### LINKED ISSUE #5302: scylla setup get stuck in an air gapped system
It looks like a repetition to Issue #4462, but now in 3.1.
*Installation details*
Scylla version (or git commit hash):3.1.1-0.20191024.f32ec885c
Cluster size:4 
OS (RHEL/CentOS/Ubuntu/AWS AMI): Centos 7.7
The following code now moved to: /usr/lib/scylla/libexec/scylla_setup
Causing Scylla_setup to halt forever when the nodes are not connected to the internet, it seems that the housekeeping is insisting on understanding if it is the latest version.

```
 if len(cur_version) > 0:
        if is_debian_variant():
            new_version=out('sudo -u scylla /usr/lib/scylla/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version {} --mode i'.format(cur_version), shell=True, exception=False)
        else:
            new_version=out('sudo -u scylla /usr/lib/scylla/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/yum.repos.d/scylla*.repo\' version --version {} --mode i'.format(cur_version), shell=True, exception=False)
        if len(new_version) > 0:
            print(new_version)
    else:
        if is_debian_variant():
            new_version=out('sudo -u scylla /usr/lib/scylla/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version unknown --mode u', shell=True, exception=False)
        else:
            new_version=out('sudo -u scylla /usr/lib/scylla/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/yum.repos.d/scylla*.repo\' version --version unknown --mode u', shell=True, exception=False)
        print('A Scylla executable was not found, please check your installation {}'.format(new_version))
```

### PR DESCRIPTION
**Pull request by amnonh**
These series solves an issue with scylla_setup and prevent it from waiting forever if housekeeping cannot look for the new Scylla version.

Fixes  #5302
It should be backported to versions that support offline installations.

### CODE REVIEW

**Comment by slivne** [comment]
@penberg ping

**Comment by slivne** [comment]
@penberg ping

**Comment by slivne** [comment]
@penberg ping

File: dist/common/scripts/scylla_setup [hunk 1]
```diff
@@ -283,16 +283,16 @@ if __name__ == '__main__':
         cur_version=out('scylla --version', exception=False)
         if len(cur_version) > 0:
             if is_debian_variant():
-                new_version=out('sudo -u scylla {}/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version {} --mode i'.format(scriptsdir(), cur_version), shell=True, exception=False)
+                new_version=out('sudo -u scylla {}/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version {} --mode i'.format(scriptsdir(), cur_version), shell=True, exception=False, timeout=60)
```
Comment (penberg) [inline dist/common/scripts/scylla_setup]: Can you please extract `timeout` into a local variable or a constant so if we decide to change 60 to something else, we don't need to scan the whole file?

**Comment by penberg** [comment]
@amnonh I switched merge target to `next`.

**Comment by slivne** [comment]
@amnonh  ping

File: dist/common/scripts/scylla_setup (hunk 1 above)
Comment (amnonh) [reply dist/common/scripts/scylla_setup]: @penberg thanks for the review, I have forced-pushed a newer version

File: dist/common/scripts/scylla_setup [hunk 2]
```diff
@@ -30,7 +30,7 @@ import stat
 from scylla_util import *
 
 interactive = False
-
+HOUSEKEEPNG_TIMEOUT = 60
```
Comment (penberg) [inline dist/common/scripts/scylla_setup]: s/HOUSEKEEPNG_TIMEOUT/HOUSEKEEPING_TIMEOUT/g

**Comment by amnonh** [comment]
@penberg thanks, forced push a new version

**Comment by penberg** [comment]
Thanks @amnonh, will merge as soon as next settles down.

**Pull request merged**